Library/Formula/cyassl.rb

```diff
@@ -63,6 +79,12 @@ def install
     system "./autogen.sh"
     system "./configure", *args
     system "make"
-    system "make install"
+    system "make", "check" if build.with? "check"
+    system "make", "install"
+  end
+
+  test do
+    # TODOS: Add a library test to accompany the binary test.
```

Comment:
I wrote up two tests last night and I somehow ran into a crashes, so I pushed a binary test instead which worked. I'll either push a library test to go with it or remove the line :).